modules/sdk/project-templates/project-templates-workspace/src/main/resources/archetype-resources/gradle.properties

```diff
@@ -1,161 +1,15 @@
-    # download. (e.g., https://releases-cdn.liferay.com/portal...)
-    #
-    #liferay.workspace.bundle.url=https://releases-cdn.liferay.com/portal/7.3.5-ga6/liferay-ce-portal-tomcat-7.3.5-ga6-20200924034643403.tar.gz
-
-    #
-    # Set this to true to configure Liferay CDN as the default repository in the
-    # root project. The default value is true.
-    #
-    #liferay.workspace.default.repository.enabled=true
-
-    #
-    # Set this property to override the default setting provided by
-    # "liferay.workspace.product". Set the Liferay Portal Docker image to create
-    # your container from. The  default value points to Liferay Portal CE 7.3
-    # GA6.
-    #
-    #liferay.workspace.docker.image.liferay=liferay/portal:7.3.5-ga6
-
-    #
-    # Set the environment with the settings appropriate for current development.
-    # The "configs" folder is used to hold different environments in the same
-    # workspace. You can organize environment settings and generate an
-    # environment installation with those settings. There are five environments:
-    # common, dev, docker, local, prod, and uat. The default value is "local".
-    #
-    #liferay.workspace.environment=local
-
-    #
-    # Set the folder that contains all Ext OSGi modules and Ext plugins. The
-    # default value is "ext".
-    #
-    #liferay.workspace.ext.dir=ext
-
-    #
-    # Set the folder that contains the Liferay bundle downloaded from the
-    # "liferay.workspace.bundle.url" property. The default value is "bundles".
-    #
-    #liferay.workspace.home.dir=bundles
-
-    #
-    # Set this to true to configure Liferay CDN as the default repository for
-    # module/OSGi projects. The default value is true.
-    #
-    #liferay.workspace.modules.default.repository.enabled=true
-
-    #
-    # Set the folder that contains all module projects. The default value is set
-    # to "modules". Set to "*" to search all subdirectories.
-    #
-    liferay.workspace.modules.dir=modules
-
-    #
-    # Set this to true to compile the JSP files in OSGi modules and have them
-    # added to the distributable Zip/Tar. The default value is false.
-    #
-    #liferay.workspace.modules.jsp.precompile.enabled=false
-
-    #
-    # Set this property to "yarn" to build Node.js-style projects using Yarn.
-    # The default value is npm.
-    #
-    #liferay.workspace.node.package.manager=npm
-
-    #
-    # Set the folder that contains the Plugins SDK environment. The default
-    # value is "plugins-sdk".
-    #
-    #liferay.workspace.plugins.sdk.dir=plugins-sdk
-
-    #
-    # Set the "liferay.workspace.product" to set the
-    # "app.server.tomcat.version", "liferay.workspace.bundle.url",
-    # "liferay.workspace.docker.image.liferay", and
-    # "liferay.workspace.target.platform.version" that matches your Liferay
-    # Product Version. To override each of these settings, set them
-    # individually.
-    #
-
+liferay.workspace.modules.dir=modules
```

Comment:
hey @lawrence-lee we need to make sure that when our "blade init" generates the product key into the file, that this new "slimmer version" of gradle.properties is correctly edited.

Reply:
Will do.

Reply:
I guess our existing project template tests would test this right?

Reply:
Yes. and i think the gradleTests will test the wildcard scenario.